### PR TITLE
Statement transaction

### PR DESCRIPTION
**Pull request by SteveOye**

### CODE REVIEW

File: lib/flutter_periculum.dart [hunk 1]
```diff
@@ -4,7 +4,9 @@ import 'dart:convert';
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:flutter_periculum/models/AffordabilityResponse.dart';
-import 'package:flutter_periculum/models/ExistingStatment.dart';
+import 'package:flutter_periculum/models/CreditScoreResponse.dart';
+import 'package:flutter_periculum/models/Statement.dart';
```
Comment (zabeehpericulum) [inline lib/flutter_periculum.dart]: I think we should be consistent and call the file and the name of the model StatementResponse.

File: example/lib/main.dart [hunk 2]
```diff
@@ -174,66 +164,51 @@ class _MyAppState extends State<MyApp> {
...
+                            }
+                            setState(() {
+                              isLoading = false;
+                            });
+                          }
+                        },
+                        child: const Text(
+                          'Get Transaction analytics',
+                        )),
+                    InkWell(
```
Comment (zabeehpericulum) [inline example/lib/main.dart]: Just a little confused about this. Is this somethin in the example app that just triggers the getStatementTransactions?

Comment (SteveOye) [reply example/lib/main.dart]: This just meant for test

File: lib/flutter_periculum.dart (hunk 1 above)
Comment (SteveOye) [reply lib/flutter_periculum.dart]: The line in red means it was remove and change with the blue

File: example/lib/main.dart (hunk 2 above)
Comment (SteveOye) [reply example/lib/main.dart]: this isn't part of the sdk

**Review by zabeehpericulum — APPROVED** [review]

**Pull request merged**